includes/multi-currency/MultiCurrency.php

```diff
@@ -1184,27 +1154,106 @@ private function register_admin_scripts() {
 		);
 	}
 
+	/**
+	 * Validates the given currency code.
+	 *
+	 * @param   string $currency_code  The currency code to check validity.
+	 *
+	 * @return  string|false  Returns back the currency code in uppercase letters if it's valid, or `false` if not.
+	 */
+	public function validate_currency_code( $currency_code ) {
```

Comment:
[minor] I suggest renaming it to `validate_and_transform_currency_code` since not only validation is done

Reply:
I think *transform* means something that provides a bigger change than converting the case to match the keys in the arrays here, so I'll probably pass this. How does it sound?

Reply:
Reasonable.
Yeah, it's better to leave it than renaming to something like `validate_and_uppercase`